src/popup/views/miscEditor/component.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { SettingsType } from "@src/lib/utilities/googleStorage";
+import { FormButtons } from "@src/components/formButtons/FormButtons";
+import { sendMessageToTab } from "@src/shared/utils";
+
+export interface MessageEditorProps {
+    liveSettings: SettingsType;
+    setLiveSettings: React.Dispatch<React.SetStateAction<SettingsType>>;
+}
+
+export function MiscEditor({
+    liveSettings,
+    setLiveSettings,
+}: MessageEditorProps): JSX.Element {
+    const [isEditing, setIsEditing] = useState(false);
+    const [savedSettings, setSavedSettings] = useState<SettingsType>({
+        ...liveSettings,
+    });
+
+    return (
+        <div className="flex flex-col h-full justify-between px-3 pb-2">
+            <div
+                className={`grid grid-cols-1 gap-y-3 w-full ${
+                    !isEditing ? "animate-fade-in" : ""
+                }`}
+            >
+                <label className="flex justify-between group p-2 hover:cursor-pointer hover:bg-slate-500/20 duration-100">
```

Comment:
let's add the tailwind class `select-none` so that users can't highlight the text `Show Chat Message Buttons`
also, let's add some styling to the background and border radius values so that users can tell that they can click on the entire button